core/controllers/reader.py

```diff
@@ -311,24 +375,33 @@ def post(self, exploration_id):
 
 
 class ClassifyHandler(base.BaseHandler):
-    """Stateless handler that performs a classify() operation server-side and
-    returns the corresponding classification result, which is a dict containing
-    three keys:
-        'outcome': A dict representing the outcome of the answer group matched.
-        'answer_group_index': The index of the matched answer group.
-        'rule_spec_index': The index of the matched rule spec in the matched
-            answer group.
-    """
+    """Stateless handler that performs a classify() operation server-side."""
 
     REQUIRE_PAYLOAD_CSRF_CHECK = False
 
     @require_playable
     def post(self, unused_exploration_id):
         """Handle POST requests.
 
-        Note: unused_exploration_id is needed because @require_playable needs 2
-        arguments.
+        Args:
+            unused_exploration_id: str. Although this is not used, it is passed
+                in because it is needed by the require_playable decorator.
+
+        Returns:
+            dict. A classification result, which is a dictionary containing
+                three keys.
+                'outcome': dict. A dict representing the outcome of the answer
+                    group matched.
+                'answer_group_index': int. An index into the answer groups list
+                    indicating which one was selected as the group which this
+                    answer belongs to. This is equal to the number of answer
+                    groups if the default outcome was matched.
+                'rule_spec_index': int. An index into the rule specs list of
+                    the matched answer group which was selected that indicates
+                    which rule spec was matched. This is equal to 0 if the
+                    default outcome is selected.
         """
+        # FIXME: Document 'outcome' dict keys.
```

Comment:
Hi @mwangikinuthia, just to check, you're planning to fix this, right? (We can't submit this PR with a FIXME in it.)